src/styles/components/header.scss

```diff
@@ -0,0 +1,71 @@
+$link-color: #00acdc;
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  line-height: 100%;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo__image {
```

Comment:
Is it the separate BEM block? Or just the part of `logo` one?
Check it and move in the separate file if it's needed